src/apiClients/AirTableApiClient.ts

```diff
@@ -24,9 +35,61 @@ class AirTableApiClient {
       `interestCategories: ${interestCategories}`,
       `riverSections: ${riverSections}`,
     );
+    let organizations: Organization[] = [];
+
+    const promise: Promise<Organization[]> = new Promise((resolve, reject) => {
+      this.base('Organizations')
+        .select({
+          view: 'Grid view',
+        })
+        .eachPage(
+          (records: any[], fetchNextPage: any) => {
+            let filteredRecords: any[] = records;
```

Comment:
I sent @olivia-hong a types file that should allow us to get rid of all of these `any` times. Ideally, everything would be typed so we can take full advantage of TypeScript's features.

Reply:
On second thought @olivia-hong @madelyndubuk, typing this is probably going to be pretty challenging and blow out of scope -- I'd still like to do that work, but we'll tackle it in a future ticket.